spec/mailers/notify_mailer_spec.rb

```diff
@@ -48,21 +48,36 @@
   end
 
   describe "#new_mention_email" do
-    let(:mention) { create(:mention, user: user2, mentionable: comment) }
-    let(:email) { described_class.with(mention: mention).new_mention_email }
+    context "when mentioning in a comment" do
+      let(:comment_mention) { create(:mention, user: user2, mentionable: comment) }
+      let(:email) { described_class.with(mention: comment_mention).new_mention_email }
 
-    it "renders proper subject" do
-      expect(email.subject).to eq("#{comment.user.name} just mentioned you!")
-    end
+      it "renders proper subject and receiver" do
```

Comment:
We made it a habit to use `aggregate_failures` for specs that have more than one assertion:

```suggestion
      it "renders proper subject and receiver", :aggregate_failures do
```

It's certainly not used consistently everywhere yet but I'd like to slowly change that.